assets/src/components/admin/admin_tables.tsx

```diff
@@ -611,14 +611,31 @@ const blueBikesColumn = {
   FormCell: FormTextarea,
 };
 
+const shuttleBusInfoColumn = {
+  Header: "Shuttle Bus Info",
+  accessor: buildAppParamAccessor("shuttle_bus_info"),
+  mutator: buildAppParamMutator("shuttle_bus_info"),
+  Cell: EditableTextarea,
+  disableFilters: true,
+  FormCell: FormTextarea,
+};
+
 const PreFareV2ScreensTable = (): JSX.Element => {
   const dataFilter = ({ app_id }) => {
     return app_id === "pre_fare_v2";
   };
 
   return (
     <AdminTable
-      columns={[...v2Columns, lineMapColumn, elevatorStatusColumn, reconstructedAlertWidgetColumn, contentSummaryColumn, blueBikesColumn]}
+      columns={[
+        ...v2Columns,
+        lineMapColumn,
+        elevatorStatusColumn,
+        reconstructedAlertWidgetColumn,
+        contentSummaryColumn,
+        blueBikesColumn,
```

Comment:
Definitely doesn't need to block you right now, but a column will need to be added for this widget.